lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1369,7 +1598,19 @@ def main():
                     allow_partial_import=module.params['allow_partial_import'],
                     cluster=otypes.Cluster(
                         name=module.params['cluster']
-                    ) if module.params['cluster'] else None
+                    ) if module.params['cluster'] else None,
+                    vnic_profile_mappings=_get_vnic_profile_mappings(module)
+                    if module.params['vnic_profile_mappings'] else None,
+                    reassign_bad_macs=module.params['reassign_bad_macs']
+                    if module.params['reassign_bad_macs'] else None,
```

Comment:
this `if` should be:
```python
module.params['reassign_bad_macs'] is not None
```

because if it `reassign_bad_macs` is `False` we will send `None`.